src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -61,7 +61,17 @@ private with sharing class ADDR_Addresses_TEST {
     /*********************************************************************************************************
     * @description The list of created test Addresses.
     */
-    private static list<Address__c> listAddrT;           
+    private static list<Address__c> listAddrT;
+    
+    private static ID orgRecTypeID;
+    private static ID householdRecTypeID;
+    private static ID adminRecTypeID;
+    
+    public static void recTypesSetup() {
```

Comment:
Here is the record types issue in AA ("never" status): https://foundation--agf.na6.visual.force.com/apex/adm_userstorydetail?id=a2x800000001i8T, and the doc where we discussed this: https://docs.google.com/document/d/1IYYfMHt-5eN6uspDaG9MGHDB_PqNSVFuA8JrnHV009Q/edit. I'm open to revisiting this and trying to come up with a better design, if we think it's worth it.